ptclient/cts/spi/dx/cl/ctDxaHistory.vue

```diff
@@ -24,7 +24,12 @@
 <script>
 import ormDx from '@/models/Dx'
 export default {
-  props: ['dx'],
+  props: {
+    dx: {
+      type: ormDx,
+      default: undefined,
+    },
+  },
```

Comment:
Props should be as detailed as possible. This tells the component the type of prop is expecting (in this case an VuexORM Model), otherwise will throw an error, and the default value.

In this particular file, `dx` is never used. But this just illustrates the improvements possible.